config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "serviceUrl": "http://wasmexplorer-service.herokuapp.com/service.php",
```

Comment:
start URL with '//' to not specify a schema

(you can modify the one below too)